### PR TITLE
feat: Implement correct behavior for all input types supporting arrow up (stepUp) and arrow down (stepDown) 

### PR DESCRIPTION
**Pull request by thevladisss**
Documents [issue 29665](https://github.com/cypress-io/cypress/issues/29665)

### CODE REVIEW

File: docs/api/actions/type.mdx [hunk 1]
```diff
@@ -172,6 +172,10 @@ locale. `yyyy-MM-dd` is the format required by
 [the W3 spec](https://www.w3.org/TR/html/infrastructure.html#dates-and-times)
 and is what the input's `value` will be set to regardless of browser or locale.
 
+`{upArrow}` and `{downArrow}` can increase and decrease the day value,
+depending on "step" parameter provided. Using cy.type() with `{upArrow}` and `{downArrow}`
```
Comment (jennifer-shehane) [inline docs/api/actions/type.mdx:176]:
```suggestion
depending on "step" parameter provided. Using `.type()` with `{upArrow}` and `{downArrow}`
```

File: docs/api/actions/type.mdx [hunk 2]
```diff
@@ -187,6 +191,11 @@ locale. `yyyy-MM` is the format required by
 [the W3 spec](https://www.w3.org/TR/html/infrastructure.html#months) and is what
 the input's `value` will be set to regardless of browser or locale.
 
+
+`{upArrow}` and `{downArrow}` can increase and decrease the month value,
+depending on "step" parameter provided. Using cy.type() with `{upArrow}` and `{downArrow}`
```
Comment (jennifer-shehane) [inline docs/api/actions/type.mdx:196]:
```suggestion
depending on "step" parameter provided. Using `.type()` with `{upArrow}` and `{downArrow}`
```

File: docs/api/actions/type.mdx [hunk 3]
```diff
@@ -207,6 +216,10 @@ and is what the input's `value` will be set to regardless of browser or locale.
 
 Special characters (`{leftArrow}`, `{selectAll}`, etc.) are not permitted.
 
+`{upArrow}` and `{downArrow}` can increase and decrease the week value,
+depending on "step" parameter provided. Using cy.type() with `{upArrow}` and `{downArrow}`
```
Comment (jennifer-shehane) [inline docs/api/actions/type.mdx:220]:
```suggestion
depending on "step" parameter provided. Using `.type()` with `{upArrow}` and `{downArrow}`
```

File: docs/api/actions/type.mdx [hunk 4]
```diff
@@ -218,6 +231,10 @@ valid time in the format:
 
 Where `HH` is 00-23, `mm` is 00-59, `ss` is 00-59, and `SSS` is 000-999.
 
+`{upArrow}` and `{downArrow}` can increase and decrease the seconds value,
+depending on "step" parameter provided. Using cy.type() with `{upArrow}` and `{downArrow}`
```
Comment (jennifer-shehane) [inline docs/api/actions/type.mdx:235]:
```suggestion
depending on "step" parameter provided. Using `.type()` with `{upArrow}` and `{downArrow}`
```

File: docs/guides/references/changelog.mdx [hunk 5]
```diff
@@ -2,6 +2,14 @@
...
 ---
 
+## 13.13.0
+
+_Released 7/02/2024 (PENDING)_
+
+**Features:**
+
+- `.type({upArrow})` and `.type({downArrow})` now also works for date, month, week, time, datetime-local and range input types. Addresses [#29665](https://github.com/cypress-io/cypress/issues/29665).
+
```
Comment (MikeMcC399) [inline docs/guides/references/changelog.mdx:12]: - There should not be any text added to the Changelog here. That is done as a bulk update as part of the release process which would pull in the change added in PR https://github.com/cypress-io/cypress/pull/29678

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
The Changelog change should be removed.

File: docs/guides/references/changelog.mdx (hunk 5 above)
Comment (thevladisss) [reply docs/guides/references/changelog.mdx:12]: Got it. Will do

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
You need to remove `changelog.mdx` completely from this PR.

**Comment by thevladisss** [comment]
> You need to remove `changelog.mdx` completely from this PR.

I am having trouble getting rid of "changelog.mdx" in this PR, perhaps it is better if I create a new PR with only relevant changes?

**Comment by MikeMcC399** [comment]
@thevladisss 

> > You need to remove `changelog.mdx` completely from this PR.
> 
> I am having trouble getting rid of "changelog.mdx" in this PR, perhaps it is better if I create a new PR with only relevant changes?

To revert all changes to `changelog.mdx`, you can copy the original file into your branch. The `git` command to do this would be:

```shell
git checkout main docs/guides/references/changelog.mdx
```

**Comment by thevladisss** [comment]
In the [PR](https://github.com/cypress-io/cypress/issues/29665) fixing this [issue](https://github.com/cypress-io/cypress/issues/29665), I've pointer out that there is missing support for "Firefox" and "Webkit" browsers for this feature. 

Do you beleive this should be mentioned in the docs?  @jennifer-shehane

**Comment by thevladisss** [comment]
> @thevladisss
> 
> > > You need to remove `changelog.mdx` completely from this PR.
> > 
> > 
> > I am having trouble getting rid of "changelog.mdx" in this PR, perhaps it is better if I create a new PR with only relevant changes?
> 
> To revert all changes to `changelog.mdx`, you can copy the original file into your branch. The `git` command to do this would be:
> 
> ```shell
> git checkout main docs/guides/references/changelog.mdx
> ```

Done

**Review by jennifer-shehane — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
Waiting on https://github.com/cypress-io/cypress/pull/29678/files to merge

**Pull request merged**